pipeline/filters/kubernetes.md

```diff
@@ -427,60 +423,58 @@ And if you are in debug mode, you could see more:
+    no - Azure does not have opinion for this user.
+    ```
 
-If you have connectivity to the API server, but still "could not get meta for POD" - debug logging might give you a message with `Azure does not have opinion for this user`. Then the following `subject` may need to be included in the `fluentbit` `ClusterRoleBinding`:
+    If you have connectivity to the API server, but still `could not get meta for POD` - debug logging might give you a message with `Azure does not have opinion for this user`. Then the following `subject` might need to be included in the `fluentbit` `ClusterRoleBinding`:
 
-appended to `subjects` array:
+    appended to `subjects` array:
 
-```yaml
-- apiGroup: rbac.authorization.k8s.io
-  kind: Group
-  name: system:serviceaccounts
-```
+    ```yaml
+    - apiGroup: rbac.authorization.k8s.io
+      kind: Group
+      name: system:serviceaccounts
+    ```
 
-**Potential fix \#2: Check Kubernetes IPv6**
+  - Potential fix #2: Check Kubernetes IPv6
 
-There may be cases where you have IPv6 on in the environment and you need to enable this within Fluent Bit. Under the service tag please set the following option `ipv6` to `on` .
+    There might be cases where you have IPv6 on in the environment and you need to enable this within Fluent Bit. Under the service tag set the following option `ipv6` to `on` .
 
-**Potential fix \#3: Check connectivity to Kube\_URL**
+  - Potential fix #3: Check connectivity to `Kube_URL`
 
-By default the Kube\_URL is set to `https://kubernetes.default.svc:443` . Ensure that you have connectivity to this endpoint from within the cluster and that there are no special permission interfering with the connection.
+    By default the Kube_URL is set to `https://kubernetes.default.svc:443`. Ensure that you have connectivity to this endpoint from within the cluster and that there are no special permission interfering with the connection.
 
-### I can't see new objects getting metadata
+- You  can't see new objects getting metadata
 
-In some cases, you may only see some objects being appended with metadata while other objects are not enriched. This can occur at times when local data is cached and does not contain the correct id for the kubernetes object that requires enrichment. For most Kubernetes objects the Kubernetes API server is updated which will then be reflected in Fluent Bit logs, however in some cases for `Pod` objects this refresh to the Kubernetes API server can be skipped, causing metadata to be skipped.
+  In some cases, you might only see some objects being appended with metadata while other objects aren't enriched. This can occur at times when local data is cached and doesn't contain the correct id for the Kubernetes object that requires enrichment. For most Kubernetes objects the Kubernetes API server is updated which will then be reflected in Fluent Bit logs. In some cases for `Pod` objects this refresh to the Kubernetes API server can be skipped, causing metadata to be skipped.
```

Comment:
NB: Phrasing, capitalize ID

```suggestion
  In some cases, you might see only some objects being appended with metadata while other objects aren't enriched. This can occur when local data is cached and doesn't contain the correct ID for the Kubernetes object that requires enrichment. For most Kubernetes objects the Kubernetes API server is updated, which will then be reflected in Fluent Bit logs. In some cases for `Pod` objects, this refresh to the Kubernetes API server can be skipped, causing metadata to be skipped.
```